src/components/flex/Flex.jsx

```diff
@@ -0,0 +1,265 @@
+
+type FlexMarginsType =
+  | 'xs'
+  | 's'
+  | 'm'
+  | 'l'
+  | 'xl'
+  | 'xxl'
+  | 'xxxl'
+  | 'xxxxl';
+
+export {
+  FLEX_DIRECTION,
+  FLEX_JUSTIFY_VALUES,
+  FLEX_ALIGNMENT_VALUES,
+  SPACING_SET as FLEX_MARGINS
+};
+
+export type FlexPropsType = {
+  /**
+   * Children to be render inside of the Box. Takes no effect if **imgSrc** prop specified.
+   * @example <Flex>
+   *            children to be render here
+   *          </Flex>
+   *
+   */
+  children: ?Node,
+  /**
+   * Additional class names
+   */
+  className?: string,
+  /**
+    * component will be rendered on 100% width of a parent
+    * @example <Flex fullWidth>
+    *            component content
+    *          </Flex>
+   */
+  fullWidth?: boolean,
+  /**
+    * component will be rendered on 100% height of a parent
+    * @example <Flex fullHeight>
+    *            component content
+    *          </Flex>
+   */
+  fullHeight?: boolean,
+  /**
+    * It will set flex-shirnk to 0
+    * @example <Flex noShrink>
+    *            component content
+    *          </Flex>
+   */
+  noShrink?: boolean,
+  /**
+   * Specify flex direction
+   * @example <Flex direction="column">
+   *            component content
+   *          </Flex>
+   * @see direction=column https://styleguide.brainly.com/latest/docs/interactive.html?direction=column#flexbox
+   * @see direction=column-reverse https://styleguide.brainly.com/latest/docs/interactive.html?direction=column-reverse#flexbox
+   * @see direction=row https://styleguide.brainly.com/latest/docs/interactive.html?direction=row#flexbox
+   * @see direction=row-reverse https://styleguide.brainly.com/latest/docs/interactive.html?direction=row-reverse#flexbox
+   */
+  direction?: ?FlexDirectionType,
+  /**
+   * Specify flex justify content 
+   * @example <Flex justifyContent="space-between">
+   *            component content
+   *          </Flex>
+   * @see justifyContent=center https://styleguide.brainly.com/latest/docs/interactive.html?justifyContent=center#flexbox
+   * @see justifyContent=flex-start https://styleguide.brainly.com/latest/docs/interactive.html?ustifyContent=flex-start#flexbox
+   * @see justifyContent=flex-end https://styleguide.brainly.com/latest/docs/interactive.html?justifyContent=flex-end#flexbox
+   * @see justifyContent=baseline https://styleguide.brainly.com/latest/docs/interactive.html?justifyContent=baseline#flexbox
+   * @see justifyContent=space-between https://styleguide.brainly.com/latest/docs/interactive.html?justifyContent=space-between#flexbox
+   * @see justifyContent=space-around https://styleguide.brainly.com/latest/docs/interactive.html?justifyContent=space-around#flexbox
+   * @see justifyContent=space-evently https://styleguide.brainly.com/latest/docs/interactive.html?justifyContent=space-evently#flexbox
+   * @see justifyContent=stretch https://styleguide.brainly.com/latest/docs/interactive.html?justifyContent=stretch#flexbox
+   */
+  justifyContent?: ?FlexJustifyValuesType,
```

Comment:
same here. please make all types not-nullable if possible.

Reply:
@PinkaminaDianePie done